### PR TITLE
(User Story 1) Display a photo grid and Front-end test coverage

### PR DESCRIPTION
**Pull request by eLeiLei**
Added components and CSS styling for a photo grid of 5 images.
Added Jest/Enzyme tests for photo components.

### CODE REVIEW

File: spec/frontend.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  test('links trigger an ajax call to getPhotosById', () => {
+    const wrapper = shallow(<PhotoBanner />);
+    const spy = jest.spyOn($, 'ajax');
+    wrapper.instance().getPhotosById();
+    expect(spy).toBeCalledWith({
+      method: 'GET',
+      url: '/api/photos/5',
+    });
+    expect(spy).toHaveBeenCalled();
+    spy.mockRestore();
```
Comment (Implausibility) [inline spec/frontend.test.js:43]: Interesting. I was attempting to access the method that contained my API call, not to spy on the caller itself.

File: spec/frontend.test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  });
+});
+
+describe('PhotoGrid Test Suite', () => {
+  test('should not find text that is not on the page', () => {
+    const photos = [{ url: 'https://japaneseanimeinfo.up.seesaa.net/image/401-3e556.jpg' }];
+    const wrapper = shallow(<PhotoGrid photos={photos} />);
+    expect(wrapper.find('text that is not on the page')).toHaveLength(0);
```
Comment (Implausibility) [inline spec/frontend.test.js:51]: What is this actually looking for? If nothing is marked as 'text that is not on the page', it's going to have a length of zero every time.

File: client/components/PhotoGrid.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function PhotoGrid({ photos }) {
+  // Load up to 5 photos for photo cover
+  const maxPhotos = photos.length < 5 ? photos.length : 5;
+  const coverPhotos = [
+    <div className="cover-item cover-item-main" key="0">
```
Comment (Implausibility) [inline client/components/PhotoGrid.jsx:8]: Do you have anything that will set these CSS names to something local? Styled Components or CSS Modules?

We need to be careful to not cross-contaminate our CSS namespaces.

Comment (eLeiLei) [reply client/components/PhotoGrid.jsx:8]: I have a ticket open to investigate Style Components. Thanks

File: spec/frontend.test.js (hunk 1 above)
Comment (eLeiLei) [reply spec/frontend.test.js:43]: While looking for a means to access the method, I came across spy. I'll want to return to this test case and investigate more to see how to improve this.

File: spec/frontend.test.js (hunk 2 above)
Comment (eLeiLei) [reply spec/frontend.test.js:51]: I could likely have taken this out. In writing these initial test cases, I wanted to ensure the next result was not a false positive. Testing my test case, if you will.

**Comment by eLeiLei** [comment]
I don't think Styled Components and CSS Modules are a requirement for the project so long as we didn't have conflicts due to overlap, and if we encountered overlaps, we'd need to resolve them. That said, I had planned to look into Styled Components and will take the suggestion of CSS Modules as an alternative. Thanks!

**Pull request merged**